### DISCUSSION TITLE
GOMAXPROCS 与容器的相处之道 | 高策

### DISCUSSION THREAD

**Discussion by giscus** [gaocegege/blog-comments#4] [General]
# GOMAXPROCS 与容器的相处之道 | 高策

GOMAXPROCS 在 Kubernetes 和 Docker 中的表现和原理分析

http://gaocegege.com/Blog/maxprocs-cpu

**Comment by gaocegege** [comment]
还有一种思路是用 LXCFS 来隔离

**Comment by grzhan** [comment]
19年看过这篇文章，没想到 2024 年这个问题还在： https://github.com/golang/go/issues/33803

**Reply by gaocegege** [reply]
大道至简

**Reply by grzhan** [reply]
笑死

**Comment by rudeigerc** [comment]
终于在 Go 1.25 里完结了 [golang/go#73193](https://github.com/golang/go/issues/73193)

**Reply by gaocegege** [reply]
感觉十年过去了）
